src/controllers/dashboard/general.js

```diff
@@ -15,11 +15,27 @@ import alert from '../../components/alert';
         page.querySelector('#txtServerName').value = systemInfo.ServerName;
         page.querySelector('#txtCachePath').value = systemInfo.CachePath || '';
         page.querySelector('#chkQuickConnectAvailable').checked = config.QuickConnectAvailable === true;
+        page.querySelector('#valDummyChapterDuration').value = config.DummyChapterDuration;
+        page.querySelector('#valDummyChapterCount').value = config.DummyChapterCount;
         $('#txtMetadataPath', page).val(systemInfo.InternalMetadataPath || '');
         $('#txtMetadataNetworkPath', page).val(systemInfo.MetadataNetworkPath || '');
         $('#selectLocalizationLanguage', page).html(languageOptions.map(function (language) {
             return '<option value="' + language.Value + '">' + language.Name + '</option>';
         })).val(config.UICulture);
+        page.querySelector('#txtChapterImageResolution').value = config.ChapterImageResolution;
+        $('#txtChapterImageResolution', page).html([
+            {name: "Match Source", value: "MatchSource"}, 
```

Comment:
If you run `npx eslint --fix` it should fix the styling issues that are listed here.